src/Raven.Server/Rachis/Leader.cs

```diff
@@ -548,6 +550,18 @@ public Task<long> PutAsync(BlittableJsonReaderObject cmd)
             return tcs.Task;
         }
         
+        public ConcurrentQueue<(string node,AlertRaised error)> ErrorsList = new ConcurrentQueue<(string,AlertRaised)>();
+
+        public void NotifyAboutException(FollowerAmbassador node, Exception e)
+        {
+            var alert = AlertRaised.Create($"Node {node.Tag} encountered an error",
+                node.Status,
+                AlertType.ClusterTopologyWarning,
+                NotificationSeverity.Warning,
+                details: new ExceptionDetails(e));
+            ErrorsList.Enqueue((node.Tag,alert));
```

Comment:
What prevent a memory leak here? If we have a failed node that always generate errors?